master-cloudconfig.yaml

```diff
@@ -0,0 +1,856 @@
+#cloud-config
```

Comment:
It is super awesome that you separated the cloudconfigs from the k8s-vm repo. Having them separated is the right thing to do IMO. I only wonder why they are here inside the AWS operator. We want to have the cluster operators for different platforms share as much dependencies as they can. The KVM operator right now used the k8s-vm repo, which I do not like. I think having some repository ala `clustercloudconfig` would be nice. The responsibility of this library should be obtaining and managing to render the cloud configs. This library is then imported by the all of our operators. The KVM operator would benefit from that as well. Remember that all of our operators should boot CoreOS via these cloud configs.